src/components/UseCases/index.tsx

```diff
@@ -0,0 +1,102 @@
+import cn from 'classnames';
+import * as React from 'react';
+
+import { Container, IContainer } from '../Container';
+import { Icon, IconProp } from '../Icon';
+import { Link } from '../Link';
+import { Section } from '../Section';
+
+export interface IUseCases {
+  bg?: string;
+}
+export const UseCases: React.FunctionComponent<IUseCases> = (bg, { ...sectionProps }) => {
```

Comment:
Looks like you're [not passing any props in](https://github.com/stoplightio/stoplight.io/pull/1240/files#diff-1a1dca2b9464b306eda3cc343574f4faR27) so suggest removing these props until we have a use for them

```suggestion
export const UseCases = () => {
```

Reply:
Deleted this component. Resolving.